### PR TITLE
Return queue length with metadata

### PR DESCRIPTION
**Pull request by leafnode**
Added feature from the queue API: returning queue length with the metadata.

### CODE REVIEW

**Comment by arafato** [comment]
This is great @leafnode , thanks a lot! Will merge tomorrow when I'm back in coding mode ;-)

File: lib/actions/queue/GetQueueMetadata.js [hunk 1]
```diff
@@ -10,11 +10,12 @@ class GetQueueMetadata {
     process(request, res) {
         const queue = QueueManager.getQueueAndMessage({ queueName: request.queueName }).queue,
             metaProps = queue.metaProps,
+            queueLength = queue.getLength(),
             response = new AzuriteQueueResponse();
-        response.addMetaProps(metaProps);
+        response.addMetaProps(metaProps, queueLength);
```
Comment (arafato) [inline lib/actions/queue/GetQueueMetadata.js]: @leafnode can you please use response.addHttpProperty('x-ms-approximate-messages-count', queueLength);

File: lib/model/queue/AzuriteQueueResponse.js [hunk 2]
```diff
@@ -17,11 +17,12 @@ class AzuriteQueueResponse {
         }
     }
 
-    addMetaProps(metaProps) {
+    addMetaProps(metaProps, queueLength) {
         Object.keys(metaProps).forEach((key) => {
             this.addHttpProperty(`x-ms-meta-${key}`, metaProps[key]);
         });
+        this.addHttpProperty(`x-ms-approximate-messages-count`, queueLength);
```
Comment (arafato) [inline lib/model/queue/AzuriteQueueResponse.js]: @leafnode and remove this line and switch back to original signature? Thanks!

File: lib/actions/queue/GetQueueMetadata.js (hunk 1 above)
Comment (leafnode) [reply lib/actions/queue/GetQueueMetadata.js]: Sure! Actually, adding more parameters to the signature looked bad, but I wanted to keep this in one place and didn't have much time for deeper refactoring. But if adding it in another line is fine - I'm good with it.

**Comment by arafato** [comment]
Great, thanks a lot @leafnode ! Just merged and will be publishing as v1.8.1 soon.

**Comment by laMoradaPostrera** [comment]
Cobos Franco, inc.

**Pull request merged**